sdk/python/kfp/v2/components/python_op.py

```diff
@@ -0,0 +1,1121 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = [
+    'create_component_from_func',
+    'func_to_container_op',
+    'func_to_component_text',
+    'default_base_image_or_builder',
+    'InputUri',
+    'InputPath',
+    'InputTextFile',
```

Comment:
Can we not expose `InputTextFile`/`InputBinaryFile` and their output counterparts for now before they're supported?

Reply:
IIUC, these could have been supported. They just translate into `{inputPath:}` placeholder.

Reply:
> They just translate into {inputPath:} placeholder.

This was a bit of surprising... Because when used with Python function component, `InputTextFile` actually indicates that system should pass a text stream object to the function, instead of a local path or URI.

Do you mind confirming what's the V1 behavior when used with ComponentSpec? Thanks!

Reply:
NVM, I just realized this is for Python function component, but not component spec. Lol

Reply:
Though they are for Python function component, I tried compiling such a function into yaml, and they resulted a component yaml definition with `String` typed input referenced with `{inputPath: }` placeholder. This violates our current rule that parameter input cannot be used with `{inputPath: }` placeholder. 

I'll remove these from the cleanup PR so that we don't expose unsupported features.